src/utils/constants.js

```diff
@@ -0,0 +1 @@
+export const NEXT_PUBLIC_MEASUREMENT_ID = "UA-235327300-1";
```

Comment:
This could be an environment variable

Reply:
Yes, I wanted this to be an env variable but wasn't sure how the deployment works on the WP side and if we can add the variable over there. That's why kept it as a const.